src/Categories/Object/Zero.agda

```diff
@@ -34,6 +35,12 @@ record Zero : Set (o ⊔ ℓ ⊔ e) where
  !-unique₂ : ∀ {A} (f g : zero ⇒ A) → f ≈ g
  !-unique₂ f g = ⟺ (!-unique f) ○ !-unique g
 
+ zero-∘ˡ : ∀ {X Y Z} → (f : Y ⇒ Z) → f ∘ zero⇒ {X} ≈ zero⇒
```

Comment:
didn't realize this file. the same principle should apply here. we can use `IsTerminal` and `IsInitial`.